flowz/channels/core.py

```diff
@@ -10,14 +10,39 @@
 
 from flowz import util
 
+
 class ChannelDone(Exception):
     """
     Exception throw when trying to access a completed channel.
 
     This is akin to StopIteration, but channel-oriented.  We can't
     use StopIteration because `tornado.gen.coroutine` swallows those.
     """
-    pass
+
+    def __init__(self, *args, **kwargs):
```

Comment:
Hmm... yes, the __init__() was added for debugging. That can go.

I had a basic assumption that you would probably want to clean up this PR, rather than take it as is. But I meant to remove the __init__.